eaf.el

```diff
@@ -300,6 +303,60 @@ Try not to modify this alist directly. Use `eaf-setq' to modify instead."
 Any new app should add the its name and the corresponding
 keybinding variable to this list.")
 
+
+(defvar eaf--bookmark-link nil)
```

Comment:
This should be defvar-local ?

Reply:
Here is my comment from above:

Not sure about this one, the variable gets set before making a bookmark and there is no need to keep it buffer local because it just records the last bookmark link used. We always need the get a fresh value from the python side before using it.

Reply:
Thinking about it, maybe we should give it a more general name because we might have other purposes to get values from the python side. This is only for transmitting some info about current state on python side so maybe we should call it something like `eaf--python-state-val`?

Reply:
If you just want a temp var, you can check ```new_buffer``` interface define in eaf.py

You can return temp variable by DBus method, just need define out_signature="s" in Dbus header, mean return a string value from Dbus method.

Reply:
I did not knew that but that sound like something I want to do ;) But I have no idea how to do that from what you said. Could you give a more detailed example what I would need to do. Here is how the value is currently retrieved:

    def set_bookmark(self):
        url = self.buffer_widget.web_page.executeJavaScript("window.location.href;")
        self.buffer_widget.set_emacs_var.emit("eaf--browser-current-url", url)

How to do that in a way you described?